project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1366 @@
+      - &1_electric_resistance_heating_heating_fuel
+        option: Heating Fuel|Electricity
+        apply_logic:
+          - not: Heating Fuel|None
+
+      - &2_electric_resistance_heating_heating_type_and_fuel_boiler 
+        option: HVAC Heating Type And Fuel|Electricity Electric Boiler
+        apply_logic: &apply_logic_fuel_boiler
+          - or: 
+            - HVAC Heating Efficiency|Fuel Boiler, 76% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 80% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 90% AFUE      
+      - &3_electric_resistance_heating_heating_type_and_fuel_furnace
+        option: HVAC Heating Type And Fuel|Electricity Electric Furnace
+        apply_logic: &apply_logic_fuel_furnace
+          - or: 
+            - HVAC Heating Efficiency|Fuel Furnace, 60% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 76% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 80% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 92.5% AFUE    
+      - &4_electric_resistance_heating_heating_type_and_fuel_wall_floor_furnace
+        option: HVAC Heating Type And Fuel|Electricity Electric Wall Furnace
+        apply_logic: &apply_logic_fuel_wall_floor_furnace
+          - or: 
+            - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+            - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+      - &5_electric_resistance_heating_heating_type_and_fuel_shared_heating
+        option: HVAC Heating Type And Fuel|Electricity Shared Heating
+        apply_logic:
+          - HVAC Heating Efficiency|Shared Heating 
+
+      - &6_electric_resistance_heating_heating_eff_boiler 
+        option: HVAC Heating Efficiency|Electric Boiler, 100% AFUE
+        apply_logic: *apply_logic_fuel_boiler      
+      - &7_electric_resistance_heating_heating_eff_fuel_furnace
+        option: HVAC Heating Efficiency|Electric Furnace, 100% AFUE
+        apply_logic: *apply_logic_fuel_furnace      
+      - &8_electric_resistance_heating_heating_eff_wall_floor_furnace
+        option: HVAC Heating Efficiency|Electric Wall Furnace, 100% AFUE
+        apply_logic: *apply_logic_fuel_wall_floor_furnace
+
+      - &9_electric_resistance_hvac_shared_eff_shared_heating_only
+        option: HVAC Shared Efficiencies|Boiler Baseboards Heating Only, Electricity
+        apply_logic:
+          - HVAC Heating Efficiency|Shared Heating
+          - not: HVAC Cooling Efficiency|Shared Cooling
+      - &10_electric_resistance_hvac_shared_eff_shared_heating_cooling
+        option: HVAC Shared Efficiencies|Fan Coil Heating and Cooling, Electricity
+        apply_logic:
+          - HVAC Heating Efficiency|Shared Heating
+          - HVAC Cooling Efficiency|Shared Cooling
+
+    #### 1.2 Federal Minimum Heat Pump With Electric Backup, 2 options  ####
+      - &1_fed_min_ashp_ducted
+        option: HVAC Heating Efficiency|ASHP, SEER 15.05, 8.82 HSPF, Max Load # same as base option
+        apply_logic:
```

Comment:
*logic_ducted_hvac_no_ashp

Reply:
try to use this reference at the first time. But got an error when validate the yml file. It looks like the logic that we referred should be in a same upgrade measure  
![image](https://github.com/NREL/resstock/assets/126117333/55292f9d-09ba-408b-94e9-358a54aef36e)

Reply:
Could be a question for Rajendra